apps/docs/content/components/RadioGroup/index.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Radio Group
+description: Radio group is used to render a short list of mutually exclusive options.
+---
+
+import ComponentLayout from '../../../src/layouts/component-layout'
+export default ComponentLayout
+
+## Usage
+
+The `RadioGroup` family of components provide a way to group related [Radio](/components/Radio) components together with supporting content such as labels, captions, and validation messages. It creates a semantic fieldset structure that helps maintain proper form hierarchy and accessibility.
+
+> **Important**: When using Radio components within a RadioGroup, ensure you provide the same `name` prop to each Radio. This ensures they are correctly grouped and only one option can be selected at a time.
```

Comment:
> **Important**: When using Radio components within a radio group, ensure you provide the same `name` prop to each Radio. This ensures they are correctly grouped and only one option can be selected at a time.

Reply:
I've just replaced this file with a (near) copy of PRC, lmk what you think